static/templates/404.html

```diff
@@ -9,5 +9,10 @@ <h1>Not Found</h1>
     <p>
       The requested URL "{{url}}" was not found on this server.
     </p>
+    {{#details}}
+    <p>
+      Make a full text search for "<a href="{{root}}/{{searchPattern}}">{{details}}</a>"
+    </p>
+    {{/details}}
```

Comment:
My understanding of [@mgautierfr's comment](https://github.com/kiwix/kiwix-lib/pull/465#pullrequestreview-614317023) was that all this extra stuff must be passed as `{{details}}`.

Reply:
@veloman-yunkan Understood. I'll do it as was asked.